uk.jnthn.backgammony.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "uk.jnthn.backgammony",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dmd"
+    ],
+    "command": "backgammony",
+    "modules": [
+        {
+            "name": "gnubg",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./configure",
+                "make datadir=/app/share",
+                "install -D gnubg /app/bin/gnubg",
+                "install -D gnubg.wd /app/share/gnubg/gnubg.wd",
+                "install -D gnubg.weights /app/share/gnubg/gnubg.weights"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "ftp://ftp.gnu.org/gnu/gnubg/gnubg-release-1.06.002-sources.tar.gz",
+                    "sha256": "ce1b0b0c1900717cc598032a14cf8c0ee60faf24d84368b39922c0102983bc87"
+                }
+            ]
+        },
+        {
+            "name": "backgammony",
+            "buildsystem": "simple",
+            "build-options": {
+                "append-path": "/usr/lib/sdk/dmd/bin",
+                "build-args": ["--share=network"]
+            },
+            "build-commands": [
+                "dub build",
+                "install -D backgammony /app/bin/backgammony",
+                "install -d \"/app/share/applications\"",
+                "install -D resources/linux/uk.jnthn.backgammony.desktop /app/share/applications",
+                "install -d \"/app/share/icons/hicolor/512x512/apps\"",
+                "install -D resources/icon.png /app/share/icons/hicolor/512x512/apps/uk.jnthn.backgammony.png",
+                "install -d /app/share/metainfo",
+                "install -D resources/linux/uk.jnthn.backgammony.appdata.xml /app/share/metainfo/uk.jnthn.backgammony.metainfo.xml"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/jonathanballs/backgammon.git",
+                    "branch": "master"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
```

Comment:
Can you move it above `modules` array? It's a nitpick, but improves readability a lot.